build.sh

```diff
@@ -101,13 +101,23 @@ if [[ $CLEAN_BUILD == 'YES' ]]; then
 fi
 mkdir -p ${BUILD_DIR} && cd ${BUILD_DIR}
 
-# If INSTALL_PREFIX is not empty; install at INSTALL_PREFIX
-[[ -n "${INSTALL_PREFIX:-}" ]] && CMAKE_OPTS+=" -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX}"
-
 # activate tests based on if this is cloned within the global-workflow
 WORKFLOW_BUILD=${WORKFLOW_BUILD:-"OFF"}
 CMAKE_OPTS+=" -DWORKFLOW_TESTS=${WORKFLOW_TESTS:-${WORKFLOW_BUILD}}"
 
+# If this is a workflow build, set INSTALL_PREFIX to Global Workflow home directory
+if [[ $WORKFLOW_BUILD == 'ON' ]]; then
```

Comment:
I feel it would simplify the overall logic if we perform `make install` regardless of whether this is a workflow build or not? And also stop allowing the user to choose the install location and hardcode it to either $HOMEgfs or the top level of GDASapp. Would that work?